django_xworkflows/models.py

```diff
@@ -65,7 +67,11 @@ def __get__(self, instance, owner):
         from a class, return the workflow.
         """
         if instance:
-            return instance.__dict__.get(self.field.name, self.field.workflow.initial_state)
+            if self.deferred_attr:
+                # use the DeferredAttribute descriptor to fetch the value
+                return self.deferred_attr.__get__(instance, owner)
+            default = self.field.to_python(self.field.workflow.initial_state)
+            return instance.__dict__.get(self.field.name, default)
```

Comment:
@rbarrois : shouldn't we raise here by the way ?
Actually I do not see the use case where the default value would be used, and this hid the bug fixed in the previous commit.
Tests still pass if I replace `.get` by `[]`.